h1b/src/hil/flash/h1b_hw.rs

```diff
@@ -261,7 +261,11 @@ impl super::hardware::Hardware<'static> for H1bHw {
 
 	fn set_transaction(&self, offset: usize, size: usize) {
 		use self::TransactionParameters::{Offset,Size};
-		self.transaction_parameters.write(Offset.val(offset as u32) + Size.val(size as u32));
+		// The offset is relative to the beginning of the flash module.
+		// TODO(jrvanwhy): Assumes the read is from the second flash bank.
+		self.transaction_parameters.write(
+			Offset.val(offset as u32 - 128 * 512)  // The first bank is 128 * 512 words
```

Comment:
Would it make sense to define this as a const earlier in the file?

Reply:
512 appears in two places so I gave it its own constant. I added an explanation for the 128, as it does not appear elsewhere.